pkg/R/sparkR.R

```diff
@@ -23,7 +23,7 @@ connExists <- function(env) {
 
 # Stop the Spark context.
 # Also terminates the backend this R session is connected to
-sparkR.stop <- function(env) {
+sparkR.stop <- function(env = .sparkREnv) {
```

Comment:
Don't add the default parameter. sparkR.stop is a finalizer that must be passed a env parameter by GC.